### PR TITLE
feat(profiling): add profiles by percentile chart to profile summary

### PR DESCRIPTION
**Pull request by eliashussary**
## Summary
Adds `<ProfileCharts/>` to the Profile Summary page; more specifically we show "Profiles by Percentile".

image

### CODE REVIEW

File: static/app/views/profiling/landing/profileCharts.tsx [hunk 1]
```diff
@@ -108,21 +118,26 @@ export function ProfileCharts({query, router, selection}: ProfileChartsProps) {
               },
               {
                 top: '32px',
-                left: '52%',
+                left: hideCount ? '24px' : '52%',
```
Comment (eliashussary) [inline static/app/views/profiling/landing/profileCharts.tsx:121]: this is kind of a lazy hack... 

we can remove all the code paths that adds the `count()` series and supplies it to the `<AreaChart />` which requires changing many props... or we can just visually hide the first chart, which is what we're doing here.

Comment (JonasBa) [reply static/app/views/profiling/landing/profileCharts.tsx:121]: Those two charts probably should have been split in the first place and just receive data as props, but this works

File: static/app/views/profiling/profileSummary/content.tsx [hunk 2]
```diff
@@ -101,6 +104,7 @@ function ProfileSummaryContent(props: ProfileSummaryContentProps) {
 
   return (
     <Layout.Main fullWidth>
+      <ProfileCharts query={props.query} router={router} hideCount />
```
Comment (JonasBa) [inline static/app/views/profiling/profileSummary/content.tsx:107]: Why not useRouter inside the charts directly?

Comment (eliashussary) [reply static/app/views/profiling/profileSummary/content.tsx:107]: we can, didn't really cross my mind haha i just blindly fixed the compiler errors 😅

Comment (eliashussary) [reply static/app/views/profiling/profileSummary/content.tsx]: fixed this up

**Review by Zylphrex — APPROVED** [review]

**Pull request merged**